### ISSUE TITLE
Set different volumes to differents souns

### ISSUE THREAD

**Issue by JoanIzquierdo** [jplayer/jPlayer#393]
Is it possible to modify the volume of different instances independently?

**Comment by MartinDawson** [comment]
Yes, very possible. You should post your question to http://stackoverflow.com/questions/tagged/jplayer with your code sample instead of here.
